pkg/irreparabledb/irreparabledb_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparabledb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/internal/testcontext"
+)
+
+func TestIrreparableRepository(t *testing.T) {
+	//testing variables
+	SegmentInfo := RemoteSegmentInfo{
```

Comment:
`SegmentInfo` -> `segmentInfo := &RemoteSegmentInfo`... then you can shorten the code below.

Reply:
done